bots/tests-invoke

```diff
@@ -373,8 +373,13 @@ class PullTask(object):
 
 
         env = os.environ.copy()
-        if "TEST_DATA" in env:
-            del env["TEST_DATA"]
+        # we can only use the cache for master; branches still use the old
+        # test/images/ symlink farm which we must not destroy
+        if opts.rebase:
```

Comment:
Master also sets opts.rebase. It's just set to 'master'. 

But @petervo is probably right. We should check for the presence of test/images/ in git on the given branch.